controller.go

```diff
@@ -860,9 +860,9 @@ curl -sLO https://raw.githubusercontent.com/inlets/inlets/master/hack/inlets-ope
 export AUTHTOKEN="` + authToken + `"
 export REMOTETCP="` + remoteTCP + `"
 export IP=$(curl -sfSL https://api.ipify.org)
-curl -SLsf https://github.com/inlets/inlets-pro-pkg/releases/download/0.4.0/inlets-pro-linux > inlets-pro-linux && \
-chmod +x ./inlets-pro-linux  && \
-mv ./inlets-pro-linux /usr/local/bin/inlets-pro
+curl -SLsf https://github.com/inlets/inlets-pro-pkg/releases/download/0.4.3/inlets-pro > inlets-pro && \
```

Comment:
Please can we download to `/tmp/`? My bad for downloading to the wd originally

Reply:
Fixed!